### PR TITLE
Update and rename Manager.php to AbstractManager.php

### PR DESCRIPTION
**Pull request by FreeFly19**

### CODE REVIEW

File: src/Layer/Manager/AbstractManager.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Layer\Manager;
+
+/**
+ * Class AbstractManager
+ * @package Layer\Manager
+ */
+abstract class AbstractManager implements ManagerInterface
```
Comment (FreeFly19) [inline src/Layer/Manager/AbstractManager.php:9]: Abstract classes MUST be prefixed by Abstract

Comment (ahonymous) [inline src/Layer/Manager/AbstractManager.php:9]: What say the PSR?

Comment (FreeFly19) [inline src/Layer/Manager/AbstractManager.php:9]: In the fig-standards repository, is the file with naming conventions for code released by PHP-FIG (https://github.com/php-fig/fig-standards/blob/master/bylaws/002-psr-naming-conventions.md). Do we need know and use them?

Comment (ahonymous) [inline src/Layer/Manager/AbstractManager.php:9]: Good.

**Pull request merged**